synapse/lib/storm.py

```diff
@@ -2762,7 +2777,26 @@ def _print_optarg(self, names, argdef):
             cstr = ', '.join(str(c) for c in choices)
             helpstr = f'{helpstr} (choices: {cstr})'
 
-        self._printf(f'{base}: {helpstr}')
+        helplst = helpstr.split('\n')
+        if helplst and not helplst[0].strip():
+            helplst = helplst[1:]
+        min_space = min((len(ln) - len(ln.lstrip()) for ln in helplst if ln.strip()), default=0)
+
+        base_w = 32
+        wrap_w = 120 - base_w
+
+        first = helplst[0][min_space:]
+        wrap_first = self._wrap_txt(first, wrap_w)
+        self._printf(f'{base:<{base_w-2}}: {wrap_first[0]}')
+
+        for ln in wrap_first[1:]: self._printf(f'{"":<{base_w}}{ln}')
+        for ln in helplst[1:]:
+            lead_s = len(ln) - len(ln.lstrip())
+            rel_ind = lead_s - min_space
+            ind = ' ' * (base_w + rel_ind)
+            wrapped = self._wrap_txt(ln.lstrip(), wrap_w - rel_ind)
```

Comment:
I think you can use textwrap.indent to accomplish this same functionality.

Reply:
I avoided importing that library and used a more targeted function.

Reply:
I think this one is worth a small bit of text formatting logic to avoid the `textwrap` modules use of `re` as well :+1: